### PR TITLE
Align download page with the getting started page

### PR DESCRIPTION
**Pull request by mlachkar**
image

### CODE REVIEW

File: _layouts/downloadpage.html [hunk 1]
```diff
@@ -16,48 +16,52 @@
...
             </div>
+
           </div>
           <div class="step">
             <div class="number-step">2</div>
             <div class="text-step">
-              <h3>Then, install Scala:</h3>
-              <p>...either by installing an IDE such as IntelliJ, or sbt, Scala's build tool.</p>
+              <h3>Then install an IDE for Scala</h3>
+              <p>The two most popular IDEs for Scala are IntelliJ and VS code with metals</p>
```
Comment (vincenzobaz) [inline _layouts/downloadpage.html]:
```suggestion
              <p>The two most popular IDEs for Scala are IntelliJ and the VS Code editor paired with the Metals extension</p>
```

**Review by vincenzobaz — APPROVED** [review]

File: _layouts/downloadpage.html (hunk 1 above)
Comment (julienrf) [inline _layouts/downloadpage.html]:
```suggestion
              <p>The two most popular IDEs for Scala are IntelliJ and VS Code with Metals</p>
```

File: _layouts/downloadpage.html [hunk 2]
```diff
@@ -16,48 +16,52 @@
...
               <ul>
                 {% include tutorial-list.html column=1 %}
               </ul>
             </div>
-            <div class="download-sbt">
+            <div class="download-vscode">
               <div class="description">
                 <img src="/resources/img/download/arrow-right.png" alt="">
-                <p>Best if you are familiar with the command line</p>
+                <p>Best if you are familiar with Vscode</p>
```
Comment (julienrf) [inline _layouts/downloadpage.html]:
```suggestion
                <p>Best if you are familiar with VS Code</p>
```

File: _layouts/downloadpage.html [hunk 3]
```diff
@@ -16,48 +16,52 @@
...
               </div>
-              <a href="" class="btn-download" id="download-sbt-link">
+              <a href="https://code.visualstudio.com/download" class="btn-download" target="_blank">
                 <i class="fa fa-download"></i>
-                <span>Download Sbt</span>
+                <span>Download VS code</span>
```
Comment (julienrf) [inline _layouts/downloadpage.html]:
```suggestion
                <span>Download VS Code</span>
```

File: _layouts/downloadpage.html [hunk 4]
```diff
@@ -79,6 +83,9 @@ <h3>Then, install Scala:</h3>
             <div class="text-step">
               <h2>Other ways to install Scala</h2>
               <ul>
+                <li>
+                  Follow <a href="https://docs.scala-lang.org/getting-started/index.html#or-manually">the getting-started</a> for a manual installation.
```
Comment (julienrf) [inline _layouts/downloadpage.html]: It is not clear to me what “manual” installation means. I guess you mean a setup with a text editor and command-line tools (no IDE)?

File: index.md [hunk 5]
```diff
@@ -9,10 +9,10 @@ headerButtonUrl: "/what-is-scala/"
...
 # Links of the Download / API Docs sections
 gettingStarted:
-  mainTitle: "Download"
-  mainUrl: "/download"
-  subtitle: "Getting Started"
-  subtitleLink: "https://docs.scala-lang.org/getting-started.html"
+  mainTitle: "Getting Started"
+  mainUrl: "/documentation/getting-started"
+  subtitle: "download"
+  subtitleLink: "/download"
```
Comment (julienrf) [inline index.md:15]: I don’t understand this change, I’m not sure what `mainUrl` and `subtitleLink` are. We need to make sure that old URLs will still work after your changes.

File: _layouts/downloadpage.html (hunk 4 above)
Comment (julienrf) [reply _layouts/downloadpage.html]: Also, it seems that this page _is_ the getting started, right?

**Review by julienrf — COMMENTED** [review]
Interesting, I think it’s OK to remove the command-line based installation instructions and focus only on IDEs. Thanks for the work!

File: _layouts/downloadpage.html (hunk 4 above)
Comment (mlachkar) [reply _layouts/downloadpage.html]: yes, it's the getting started.

File: index.md (hunk 5 above)
Comment (mlachkar) [reply index.md:15]: For me this change is important: the goal is to give more importance to the learning process, so first the `getting-started page` then the download page. Is that ok for you! 
image

Comment (mlachkar) [reply index.md:15]: the `documentation/getting-started` is a redirect .. but I can put the complete web address

File: _layouts/downloadpage.html (hunk 4 above)
Comment (julienrf) [reply _layouts/downloadpage.html]: So, what I mean is that we are already in the “getting started” page and we talk about another “getting started” page, so I think this is a bit confusing. Furthermore, I don’t think “manual” installation is very clear.

Comment (mlachkar) [reply _layouts/downloadpage.html]: ok. I will remove this. I agree

Comment (julienrf) [reply _layouts/downloadpage.html]: I think it is useful to still keep a link to setup instructions that are not IDE-based. I just don’t think the phrasing is clear here.

File: _data/tutorials.yml [hunk 6]
```diff
@@ -1,10 +1,6 @@
...
-  url: "/documentation/getting-started-sbt-track/getting-started-with-scala-and-sbt-on-the-command-line.html"
-- title: "Building a Scala Project with IntelliJ and sbt"
-  url: "/documentation/getting-started-intellij-track/building-a-scala-project-with-intellij-and-sbt.html"
-- title: "Testing Scala with sbt and ScalaTest on the Command Line"
-  url: "/documentation/getting-started-sbt-track/testing-scala-with-sbt-on-the-command-line.html"
-- title: "Testing Scala in IntelliJ with ScalaTest"
-  url: "/documentation/getting-started-intellij-track/testing-scala-in-intellij-with-scalatest.html"
+- title: "Create a Hello-world project with sbt and IntelliJ"
+  url: "https://docs.scala-lang.org/getting-started/index.html#using-intellij"
+- title: "Create a Hello-world project with sbt and Vscode"
```
Comment (julienrf) [inline _data/tutorials.yml]:
```suggestion
- title: "Create a Scala project with sbt and VS Code"
```

File: _data/tutorials.yml [hunk 7]
```diff
@@ -1,10 +1,6 @@
...
+  url: "https://docs.scala-lang.org/getting-started/index.html#using-vscode-with-metals"
+- title: "Create a Hello world within Intellij (no command line)"
```
Comment (julienrf) [inline _data/tutorials.yml]:
```suggestion
- title: "Create a Scala project within Intellij (no command line)"
```

File: _data/tutorials.yml [hunk 8]
```diff
@@ -1,10 +1,6 @@
...
-  url: "/documentation/getting-started-intellij-track/getting-started-with-scala-in-intellij.html"
-- title: "Getting Started with Scala and sbt on the Command Line"
-  url: "/documentation/getting-started-sbt-track/getting-started-with-scala-and-sbt-on-the-command-line.html"
-- title: "Building a Scala Project with IntelliJ and sbt"
-  url: "/documentation/getting-started-intellij-track/building-a-scala-project-with-intellij-and-sbt.html"
-- title: "Testing Scala with sbt and ScalaTest on the Command Line"
-  url: "/documentation/getting-started-sbt-track/testing-scala-with-sbt-on-the-command-line.html"
-- title: "Testing Scala in IntelliJ with ScalaTest"
-  url: "/documentation/getting-started-intellij-track/testing-scala-in-intellij-with-scalatest.html"
+- title: "Create a Hello-world project with sbt and IntelliJ"
```
Comment (julienrf) [inline _data/tutorials.yml]:
```suggestion
- title: "Create a Scala project with sbt and IntelliJ"
```

File: _layouts/downloadpage.html [hunk 9]
```diff
@@ -16,48 +16,52 @@
...
-              <h3>First, make sure you have the Java 8 JDK (or Java 11 JDK) installed.</h3>
-              <p>To check, open the terminal and type:</p>
-              <p><code>java -version</code><span>(Make sure you have version 1.8 or 11.)</span></p>
-              <p><i>(If you don't have it installed, download Java from <a href="https://www.oracle.com/java/technologies/javase-jdk8-downloads.html">Oracle Java 8</a>, <a href="https://www.oracle.com/java/technologies/javase-jdk11-downloads.html">Oracle Java 11</a>, or <a href="https://adoptopenjdk.net/">AdoptOpenJDK 8/11</a>. Refer <a href="https://docs.scala-lang.org/overviews/jdk-compatibility/overview.html">JDK Compatibility</a> for Scala/Java compatiblity detail.</i></p>
+              <h3>First, install Java, Scala, and all related command-line tools.</h3>
+              <p>Download the <code>cs</code> tool and execute the <code>setup</code> command</p>
+                <div id="download-step-one">
+                  <p>Follow <a href="https://get-coursier.io/docs/cli-overview.html#install-native-launcher" target="_blank">the instructions to install the <code>cs</code> launcher</a> then run:</p>
```
Comment (julienrf) [inline _layouts/downloadpage.html]:
```suggestion
                  <p>Follow <a href="https://get-coursier.io/docs/cli-overview.html#install-native-launcher" target="_blank">the instructions to install the <code>cs</code> launcher</a>, then run:</p>
```

File: _layouts/downloadpage.html [hunk 10]
```diff
@@ -16,48 +16,52 @@
...
+              <p>(See <a href="https://get-coursier.io/docs/cli-installation">the instructions for other operating systems.</a>)</p>
             </div>
+
           </div>
           <div class="step">
             <div class="number-step">2</div>
             <div class="text-step">
-              <h3>Then, install Scala:</h3>
-              <p>...either by installing an IDE such as IntelliJ, or sbt, Scala's build tool.</p>
+              <h3>Then install an IDE for Scala</h3>
```
Comment (julienrf) [inline _layouts/downloadpage.html]:
```suggestion
              <h3>Then, install an IDE for Scala</h3>
```

**Review by julienrf — APPROVED** [review]
I have left some suggestions, but overall this looks great, thanks!

**Pull request merged**